src/game/camera.c

```diff
@@ -5210,7 +5351,11 @@ void set_camera_mode_8_directions(struct Camera *c) {
     if (c->mode != CAMERA_MODE_8_DIRECTIONS) {
         c->mode = CAMERA_MODE_8_DIRECTIONS;
         sStatusFlags &= ~CAM_FLAG_SMOOTH_MOVEMENT;
+#ifdef REONUCAM
+        s8DirModeBaseYaw = snap_to_45_degrees(s8DirModeBaseYaw);
+#else
```

Comment:
this seems weird, how do you know what s8DirModeBaseYaw is here? that variable is not used outside of 8 direction camera afaik so i don't know if this has the effect you think it does

Reply:
I think you're right and this is completely unnecessary